pathspider/sync.py

```diff
@@ -23,6 +23,8 @@ def __init__(self, worker_count, libtrace_uri, args, server_mode=False):
         super().__init__(worker_count, libtrace_uri, args, server_mode)
         self.__logger = logging.getLogger('sync')
 
+        if type(self.configurations).__name__ == 'method':
```

Comment:
Use `callable(self.configurations)` to check if it is callable instead of this hack.